{"uuid":"dc50b522-e420-461a-8558-45ddcab07413","historyId":"5e1624f7b8e94f8547a5ff91f3e82025","fullName":"com.e2e.LinkedinAutoAddReqTest.login","labels":[{"name":"package","value":"com.e2e.LinkedinAutoAddReqTest"},{"name":"testClass","value":"com.e2e.LinkedinAutoAddReqTest"},{"name":"testMethod","value":"login"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"LinkedInBot"},{"name":"subSuite","value":"com.e2e.LinkedinAutoAddReqTest"},{"name":"host","value":"WS-024"},{"name":"thread","value":"13256@WS-024.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"login","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.ExceptionInInitializerError","trace":"java.lang.ExceptionInInitializerError\r\n\tat com.xiokrius.linkedin.linkedinPages.login.<init>(login.java:13)\r\n\tat com.e2e.LinkedinAutoAddReqTest.login(LinkedinAutoAddReqTest.java:35)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:819)\r\n\tat org.testng.TestRunner.run(TestRunner.java:619)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1228)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1134)\r\n\tat org.testng.TestNG.run(TestNG.java:1101)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: java.lang.RuntimeException: Файл не найден: config.properties.contacts\r\n\tat com.xiokrius.linkedin.ConfigManager.loadProperties(ConfigManager.java:20)\r\n\tat com.xiokrius.linkedin.ConfigManager.<clinit>(ConfigManager.java:12)\r\n\t... 27 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1757422217274,"stop":1757422217310}